packages/app-explorer/src/Events.tsx

```diff
@@ -56,7 +56,7 @@ function Events ({ emptyLabel, eventClassName, events, withoutIndex, t }: Props)
               <summary>
                 {
                   event.meta && event.meta.documentation
-                    ? event.meta.documentation.join(' ')
+                    ? event.meta.documentation.join(' ').split('\\[info\\]').join(' ')
```

Comment:
I guess we can also use `replace(//g, '' )` to replace all, but yeah, in this case we will need to write code like `a.replace(/\\\[info\\\]/g,"rr")` to process the special character `[` and `]`, which make `a.replace(/\\\[info\\\]/g,"rr")` looks confuse T T. love your implement 👍